packages/paste-core/components/anchor/stories/index.stories.tsx

```diff
@@ -72,16 +73,13 @@ ShowExternalProp.story = {
 };
 
 export const CustomAnchor = (): React.ReactNode => {
+  const currentTheme = useTheme();
```

Comment:
question: why this particular change? Wouldn't passing `baseTheme="default"` with no `theme` prop have the same effect?

Reply:
This is to do with the "Stacked" or "Side by Side" view of our stories when we render them in test environments. One side it light, one side is dark, and all we're doing is rendering the same story in two views. 

So if you set a static baseTheme of default, when it's rendered in Dark theme in Stacked or Side by Side, the component is using light mode tokens, in a dark background. You have to basically dynamically inherit the theme you're rendering the story in